SystemSettings.php

```diff
@@ -354,37 +303,33 @@ public function isMysqlBackend()
     private function createBackendSetting()
     {
         return $this->makeSetting('backend', $default = 'redis', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
-            $field->title = 'Backend';
+            $field->title = Piwik::translate('QueuedTracking_BackendSettingFieldTitle');
             $field->uiControl = FieldConfig::UI_CONTROL_SINGLE_SELECT;
             $field->availableValues = array('redis' => 'Redis', 'mysql' => 'MySQL');
-            $field->inlineHelp = 'Select the backend you want to use for this feature. If you do not have any experience with Redis or it is not available on your server, we recommend to use Mysql.';
+            $field->inlineHelp = Piwik::translate('QueuedTracking_BackendSettingFieldHelp') . '</br>';
```

Comment:
@snake14 I think we need to add 1 more `<br>` to ensure the screenshot matches

Reply:
I actually added the linebreak so that the default wouldn't show on the line directly below the help text. A second isn't necessary.